dockerfiles/base/scripts/base/commands/cmd_destroy.sh

```diff
@@ -46,26 +46,14 @@ cmd_destroy() {
                                    && rm -rf /root${CHE_CONTAINER_ROOT}/instance \
                                    && rm -rf /root${CHE_CONTAINER_ROOT}/${CHE_MINI_PRODUCT_NAME}.env\""
 
-  docker_run -v "${CHE_HOST_CONFIG}":/root${CHE_CONTAINER_ROOT} \
-                alpine:3.4 sh -c "rm -rf /root${CHE_CONTAINER_ROOT}/docs \
-                               && rm -rf /root${CHE_CONTAINER_ROOT}/instance \
-                               && rm -rf /root${CHE_CONTAINER_ROOT}/${CHE_MINI_PRODUCT_NAME}.env" > /dev/null 2>&1  || true
-
   # Super weird bug.  For some reason on windows, this command has to be run 3x for everything
```

Comment:
No, I haven't created an issue.  I wasn't sure it was a docker issue for it.  We discovered this a couple months ago when we first implemented the `cmd_destroy` function.  We were seeing that we had to 3x remove stuff in that function and now it's coming up again.  I can create that issue for them.